common/src/main/java/org/apache/celeborn/common/network/TransportContext.java

```diff
@@ -55,6 +57,7 @@
 public class TransportContext {
   private static final Logger logger = LoggerFactory.getLogger(TransportContext.class);
 
+  private static final NettyLogger nettyLogger = new NettyLogger();
```

Comment:
Nit: why static?

Reply:
We can have multiple TransportContext's right ?
This logger is going to be same across all of them

Reply:
K